tests/binary.rs

```diff
@@ -19,12 +18,12 @@ mod binary {
     data
   }
 
-  fn get_tempfile_path(extension: &str) -> PathBuf {
+  fn get_tempfile_path(extension: &str) -> String {
```

Comment:
This is potentially wrong, why do you need to do this?

Reply:
To use the filepath in `Command.args(...)` (e.g. [here](https://github.com/xiph/rav1e/pull/2344/files#diff-b1aba0aa09c8f43383202cbf5085a064R54) as `&*outfile`), it needs to be `&str`. So I though doing the conversion in one place is easier than multiple times just before the `.args(...)`.

What error scenarios are there?

Reply:
**lu-zero** on irc: String cannot represent paths in the operating systems having non-utf8 locale, Command arg and args take any type that can be an OsStr for that reason

Reply:
Addressed this in https://github.com/xiph/rav1e/pull/2344/commits/e984ddbc7c0c41d76ca9128f876bc8735dbb2992

Reply:
Do not use `args()` and keep the pathbuf please.

Reply:
Okay. Also fine.